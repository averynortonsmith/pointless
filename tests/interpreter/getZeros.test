
output = [a, b] |> println

a = PtlsArray.!getZeros
b = PtlsArray.!getZeros(10)
