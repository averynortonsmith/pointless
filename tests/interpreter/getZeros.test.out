[PtlsBuiltIn, [0 0 0 0 0 0 0 0 0 0]]
